src/structures/player.ts

```diff
@@ -102,6 +102,17 @@ export default class Player {
         return name;
     }
 
+    /**
+     * @param guildID - The guild ID
+     * @returns the player's nickname in the given guild
+     */
+    getName(): string {
+        return (
+            State.client.guilds.get(this.guildID).members.get(this.id).nick ??
```

Comment:
Why retrieve them from state here, instead of passing them in? Should try to avoid global accesses in our plain classes

Reply:
If a user changes their nickname mid-game, it can become confusing who is who. Some games run for hours, which can make this worse